edx_proctoring/templates/proctored_exam/visit_exam_content.html

```diff
@@ -1,11 +1,11 @@
 {% load i18n %}
 {% if has_due_date_passed %}
-
+  <hr>
   <p>
     {% blocktrans %}
-      If you want to view your exam questions and responses, click on the button “let me see my exam”.
-      When you clicked the button, you will be taken directly to the exam content. The exam’s status will
-      still be visible to you in the left navigation pane.
+      If you want to view your exam questions and responses, click on the “Let me see my exam” button below.
```

Comment:
@chrisndodge I also have a suggestion for the "Let me see my exam" button text -> "View my exam".
If you go with this suggestion, then also modify the "To view your exam questions..." text from my comment above.